probePlugin/src/main/scala/org/virtuslab/ideprobe/handlers/RunConfigurations.scala

```diff
@@ -1,35 +1,39 @@
 package org.virtuslab.ideprobe.handlers
 
-import java.util.Collections
-import java.util.UUID
 import java.util.concurrent.CountDownLatch
+import java.util.{Collections, UUID}
+
 import com.intellij.compiler.options.CompileStepBeforeRun.MakeBeforeRunTask
-import com.intellij.execution.ExecutionManager
-import com.intellij.execution.RunManager
-import com.intellij.execution.RunnerAndConfigurationSettings
+import com.intellij.execution._
 import com.intellij.execution.application.ApplicationConfiguration
 import com.intellij.execution.executors.DefaultRunExecutor
-import com.intellij.execution.impl.RunManagerImpl
-import com.intellij.execution.impl.RunnerAndConfigurationSettingsImpl
+import com.intellij.execution.impl.{RunManagerImpl, RunnerAndConfigurationSettingsImpl}
```

Comment:
Do you have any conventions on the formatting of imports?

Reply:
Perhaps we should setup scalafmt, as these import changes are distracting